src/devices/LedDisplay/category.txt

```diff
@@ -0,0 +1,3 @@
+display
+led
```

Comment:
I don't think `led` applies here. LED was meant to be bindings like LED strip or LED driver. While this binding to have LEDs to turn on segments I think it doesn't exactly fit well. Easier way to think about it is to try to predict how changing this will affect https://github.com/dotnet/iot/tree/master/src/devices - In this case it will update [LED drivers](https://github.com/dotnet/iot/tree/master/src/devices#led-drivers) which you can probably see it doesn't fit here.

I'm fine with changing the name of this category if you think it's a bit bad and also fine if you remove this file at all since not having it produces the warning when running the tool which I always fix when updating (I always focus on the output doc making sense which was the main purpose of the categories)